src/main/java/org/usfirst/frc4904/robot/RobotMap.java

```diff
@@ -68,8 +170,48 @@
+        Component.leftWheelEncoder = new CANCoder(Port.CAN.LEFT_WHEEL_ENCODER);
+        Component.rightWheelEncoder = new CANCoder(Port.CAN.RIGHT_WHEEL_ENCODER);
+        Component.canCoderConfiguration = new CANCoderConfiguration();
+        Component.leftWheelEncoder.configAllSettings(Component.canCoderConfiguration);
+        Component.rightWheelEncoder.configAllSettings(Component.canCoderConfiguration);
+        Component.leftWheelEncoder.configFeedbackCoefficient(RobotMap.Metrics.Chassis.METERS_PER_TICK, "meters",
+                SensorTimeBase.PerSecond);
+        Component.rightWheelEncoder.configFeedbackCoefficient(RobotMap.Metrics.Chassis.METERS_PER_TICK, "meters",
+                SensorTimeBase.PerSecond);
+
+        Component.leftWheelAccelerationCap = new EnableableModifier(new AccelerationCap(Component.pdp));
+        Component.leftWheelAccelerationCap.enable();
+        Component.rightWheelAccelerationCap = new EnableableModifier(new AccelerationCap(Component.pdp));
+        Component.rightWheelAccelerationCap.enable();
+
+        Component.leftDriveA = new Motor("leftDriveA", false, Component.leftWheelAccelerationCap,
+                new CANTalonFX(Port.CANMotor.LEFT_DRIVE_A));
+        Component.leftDriveB = new Motor("rightDriveA", false, Component.rightWheelAccelerationCap,
+                new CANTalonFX(Port.CANMotor.RIGHT_DRIVE_A));
+        Component.rightDriveA = new Motor("leftDriveB", false, Component.leftWheelAccelerationCap,
+                new CANTalonFX(Port.CANMotor.LEFT_DRIVE_B));
+        Component.rightDriveB = new Motor("rightDriveB", false, Component.rightWheelAccelerationCap,
+                new CANTalonFX(Port.CANMotor.RIGHT_DRIVE_B));
+
+        Component.turnPID = new CustomPIDController(PID.Turn.P, PID.Turn.I, PID.Turn.D, PID.Turn.F, (PIDSensor) Component.navx);
+        Component.shifter = new SolenoidShifters(Port.Pneumatics.SHIFTER.buildDoubleSolenoid());
+
+        Component.chassis = new TankDriveShifting(0.0, RobotMap.Component.leftDriveA, RobotMap.Component.leftDriveB,
+                RobotMap.Component.rightDriveA, RobotMap.Component.rightDriveB, Component.shifter);
+
+        Component.splinesChassis = new SensorDrive(Component.chassis, AutoConstants.autoConstants, DriveConstants.driveConstants, Component.leftWheelEncoder, Component.rightWheelEncoder, Component.navx);
+
+        HumanInput.Driver.xbox = new CustomXbox(Port.HumanInput.XBOX_CONTROLLER);
+        HumanInput.Operator.joystick = new CustomJoystick(Port.HumanInput.JOYSTICK);
+        NetworkTables.inst = NetworkTableInstance.getDefault();
+        NetworkTables.table = NetworkTables.inst.getTable("vision");
+        NetworkTables.Vision.distanceToTarget = NetworkTables.Vision.table.getEntry("distanceToTarget");
+        NetworkTables.Vision.beta = NetworkTables.Vision.table.getEntry("beta");
+        NetworkTables.Vision.theta = NetworkTables.Vision.table.getEntry("theta");
```

Comment:
Remember to update this to x and y. Also, if we're getting different values for each camera, make sure that we're referencing a specific camera's data (maybe make different tables for each camera?)